frontend/src/types.ts

```diff
@@ -205,7 +205,7 @@ export type OdhDocument = {
     appCategory?: string; // Only set on UI side in resources section
     provider?: string;
     description: string;
-    url: string;
+    url: string | undefined;
```

Comment:
`url` is a required property in the CRD.
```suggestion
    url: string;
```